backend/src/org/commcare/resources/model/installers/CacheInstaller.java

```diff
@@ -46,24 +47,20 @@
         return cacheStorage;
     }
 
-    int cacheLocation;
+    @Override
```

Comment:
I know you didn't add this but what is the purpose of having this abstract method declaration here, when it's just overriding an abstract method of its super class?